### ISSUE TITLE
feature request: option to unblock

### ISSUE THREAD

**Issue by mixieculez** [satsukitv/twitter-block-chain#23]
Would be nice to have an ability to undo blocking.

**Comment by roycewilliams** [comment]
Definitely interested in this one - blocking-worthiness signals can be subtle. There have been multiple cases where I wanted to 'Run unblock chain' after an inadvertent block run.
